### DISCUSSION TITLE
is it possible to deploy from github action ci?

### DISCUSSION THREAD

**Discussion by ClarissaGazalaEvanthe** [invertase/globe#22] [Q&A]
is it possible to deploy from github action ci?

**Comment by lesnitsky** [comment]
You can connect a GitHub repo to Globe and it will automatically trigger deployment to prod from main branch and preview when you make a PR.

You can create a Globe project that is integrated with GitHub from https://globe.dev/new
For a project created via CLI, you can enable a GitHub integration from project settings:
- Open your project
- Go to Settings
- Click on "Git" tab
- Authorize a Globe app to have access to your repo
- Click "Connect"

We are working on making it possible to authenticate Globe CLI from CI.
